src/main/java/kunkka/command/EventCommand.java

```
@@ -0,0 +1,60 @@
+package kunkka.command;
+import kunkka.components.Event;
+import kunkka.components.KunkkaException;
+import kunkka.tasklist.Tasklist;
+
+/**
+ * Represents a command to add an event task to the task list.
+ */
+public class EventCommand extends Command {
+    
+    protected String description;
+    protected String from;
+    protected String to;
+
+    /**
+     * Constructs an EventCommand object.
+     * 
+     * @param description Description of the event task.
+     * @param from Start time of the event task.
+     * @param to End time of the event task.
+     */
+    public EventCommand(String description, String from, String to){
+        super("event");
+        this.description = description;
+        this.from = from;
+        this.to = to;
+    }
+
+    /**
+     * Executes the event command to add an event task to the task list.
+     * 
+     * @param tasks Tasklist object that stores the list of tasks.
+     */
```

Perhaps you can include `@throw KunkkaException for invalid inputs` for a more complete JavaDoc comment?

I noticed the same issue for other places too.